config/src/lib.rs

```diff
@@ -0,0 +1,137 @@
+use apache_avro::Schema;
+use lazy_static::lazy_static;
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+use std::collections::hash_map::HashMap;
+
+pub const MAINNET_CONFIG: &str = include_str!("../resources/configs/frequency.json");
+pub const ROCOCO_CONFIG: &str = include_str!("../resources/configs/frequency-rococo.json");
+
+pub type SchemaId = u16;
+
+lazy_static! {
+	pub static ref PUBLIC_KEY_SCHEMA: Schema =
+		Schema::parse_str(include_str!("../resources/schemas/public_key_schema.json")).unwrap();
+	pub static ref PUBLIC_GRAPH_CHUNK_SCHEMA: Schema =
+		Schema::parse_str(include_str!("../resources/schemas/user_public_graph_chunk.json"))
+			.unwrap();
+	pub static ref PUBLIC_GRAPH_SCHEMA: Schema =
+		Schema::parse_str(include_str!("../resources/schemas/public_graph.json")).unwrap();
+	pub static ref PRIVATE_GRAPH_CHUNK_SCHEMA: Schema =
+		Schema::parse_str(include_str!("../resources/schemas/user_private_graph_chunk.json"))
+			.unwrap();
+}
+
+/// Privacy Type of the graph
+#[derive(Clone, Copy, PartialEq, Ord, Eq, PartialOrd, Debug, Hash, Serialize, Deserialize)]
```

Comment:
Should these types be defined out side config? any intentions why here?

Reply:
In order to be able to have config manage the mapping ConnectionType <=> SchemaId, it needs to know about those enums. If we define them in the core crate, then we get a circular dependency. The other option would be to factor out into a `dsnp-graph-common` or `dsnp-graph-types` crate...

Reply:
I see, let's keep it here then.